### ISSUE TITLE
cc, replyTo issues with project-config/sync and rebuild

### ISSUE THREAD

**Issue by timkelty** [craftcms/commerce#1111]
When I  run `craft project-config/sync`,  I get an error about missing keys `cc` and `replyTo` for my commerce emails:

```
error: Undefined index: cc
[2019-11-05 00:09:43] craftcms.ERROR: yii\base\ErrorException: Undefined index: cc in /app/vendor/composer/craftcms/commerce/src/services/Emails.php:221 Stack trace: #0 /app/vendor/composer/craftcms/commerce/src/services/Emails.php(221): yii\base\ErrorHandler->handleError(8, 'Undefined index...', '/app/vendor/com...', 221, Array) #1 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(867): craft\commerce\services\Emails->handleChangedEmail(Object(craft\events\ConfigEvent)) #2 [internal function]: craft\services\ProjectConfig->craft\services\{closure}(Object(craft\events\ConfigEvent)) #3 /app/vendor/composer/yiisoft/yii2/base/Component.php(627): call_user_func(Object(Closure), Object(craft\events\ConfigEvent)) #4 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(554): yii\base\Component->trigger('updateItem', Object(craft\events\ConfigEvent)) #5 /app/vendor/composer/craftcms/commerce/src/services/OrderStatuses.php(265): craft\services\ProjectConfig->processConfigChanges('commerce.emails...') #6 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(867): craft\commerce\services\OrderStatuses->handleChangedOrderStatus(Object(craft\events\ConfigEvent)) #7 [internal function]: craft\services\ProjectConfig->craft\services\{closure}(Object(craft\events\ConfigEvent)) #8 /app/vendor/composer/yiisoft/yii2/base/Component.php(627): call_user_func(Object(Closure), Object(craft\events\ConfigEvent)) #9 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(554): yii\base\Component->trigger('updateItem', Object(craft\events\ConfigEvent)) #10 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(861): craft\services\ProjectConfig->processConfigChanges('commerce.orderS...') #11 [internal function]: craft\services\ProjectConfig->craft\services\{closure}(Object(craft\events\ConfigEvent)) #12 /app/vendor/composer/yiisoft/yii2/base/Component.php(627): call_user_func(Object(Closure), Object(craft\events\ConfigEvent)) #13 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(554): yii\base\Component->trigger('updateItem', Object(craft\events\ConfigEvent)) #14 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(942): craft\services\ProjectConfig->processConfigChanges('commerce.orderS...') #15 /app/vendor/composer/craftcms/cms/src/services/ProjectConfig.php(436): craft\services\ProjectConfig->_applyChanges(Array) #16 /app/vendor/composer/craftcms/cms/src/console/controllers/ProjectConfigController.php(87): craft\services\ProjectConfig->applyYamlChanges() #17 [internal function]: craft\console\controllers\ProjectConfigController->actionSync() #18 /app/vendor/composer/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array) #19 /app/vendor/composer/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array) #20 /app/vendor/composer/yiisoft/yii2/console/Controller.php(148): yii\base\Controller->runAction('sync', Array) #21 /app/vendor/composer/craftcms/cms/src/console/Controller.php(199): yii\console\Controller->runAction('sync', Array) #22 /app/vendor/composer/yiisoft/yii2/base/Module.php(528): craft\console\Controller->runAction('sync', Array) #23 /app/vendor/composer/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('project-config/...', Array) #24 /app/vendor/composer/craftcms/cms/src/console/Application.php(93): yii\console\Application->runAction('project-config/...', Array) #25 /app/vendor/composer/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('project-config/...', Array) #26 /app/vendor/composer/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request)) #27 /app/bin/craft(7): yii\base\Application->run() #28 {main} {"trace":[],"memory":39691384,"category":"yii\\base\\ErrorException:8","exception":"[object] (yii\\base\\ErrorException(code: 8): Undefined index: cc at /app/vendor/composer/craftcms/commerce/src/services/Emails.php:221)"} []
error Command failed with exit code 1.
```

If I add them, to my project config and run sync again, things work:

```
      cc: ''
      replyTo: ''
```

…however, when I later run `project-config/rebuild`, they are removed from my project.yaml.

Note: if I emails in those values, things seem ok. Issue happens when they are empty.

Also not sure if relevant, but this is a Craft 2 upgraded to Craft 3.

**Comment by nfourtythree** [comment]
Hi @timkelty 

This was fixed in the following commit: 4cb3fc33e3455eb3ce77987c33d721289d407e35

It will be included in the next release.

Thanks.

**Comment by timkelty** [comment]
Thanks, missed that.
